src/phases/collection.rs

```diff
@@ -126,19 +142,18 @@ fn find_unittest_base(expr: &ast::Expr) -> bool {
 fn find_unittest_class_cases(
     stmts: Vec<Stmt>,
     prefix: &str,
-    file_name: String,
-    verbose: bool,
+    _file_name: String,
+    _verbose: bool,
 ) -> Vec<String> {
     let mut cases = vec![];
     for stmt in stmts {
         match stmt {
             FunctionDef(node)
-                if node.name.starts_with(prefix)
-                    && !ignore_test::is_pytest_fixture(stmt.clone()) =>
+                if node.name.starts_with(prefix) && !ignore_test::is_pytest_fixture(&stmt) =>
             {
                 cases.push(node.name.to_string())
             }
-            _ if verbose => println!("{}: Skipping {:#?}\n\n", file_name, stmt),
+            //_ if verbose => println!("{}: Skipping {:#?}\n\n", file_name, stmt),
```

Comment:
I commented this out. Maybe `-vv` for this one since this prints the whole AST?

Reply:
yeah that's a good call, that makes sense.  fine if you want to just leave it as is for now though and we handle that in a later PR.